src/components/Scheme/ReputationFromToken.tsx

```diff
@@ -116,13 +144,15 @@ class ReputationFromToken extends React.Component<IProps, IState> {
     if (alreadyRedeemed) {
```

Comment:
Already redeemed should be based on the `redeemerAddress`, not the `accountAddress`
Looks like some nasty copy-pasting from line  101, probably worth to refactor out to one method ^_^

Reply:
good call, this was already a bug in the previous version..